stacks-common/src/libcommon.rs

```diff
@@ -31,6 +31,7 @@ pub mod address;
 
 pub mod deps_common;
 
+#[cfg(feature = "canonical")]
```

Comment:
I'm curios why this needs to be gated?

Reply:
Because bitvec.rs uses rustqlite. It seems just fine to isolate the whole module because it's not used in stacks-common or clarity.
I could also gate smaller parts of the module (only `FromSql` and `ToSql`) if you think it's better

Reply:
Yeah, let's go ahead and isolate the `FromSql` and `ToSql` implementations for `BitVec`.  There's no reason to exclude `BitVec` from a WASM build, especially since Nakamoto blocks contain one.